src/preset.js

```diff
@@ -31,7 +31,7 @@ genotet.preset.NETWORK_PARAMS_ = {
  */
 genotet.preset.EXPRESSION_PARAMS_ = {
   fileName: 'rnaseq',
-  tfaFileName: 'tfa.mat.tsv',
+  tfaFileName: 'tfa_rnaseq.mat.tsv',
```

Comment:
what's the naming rule? I think rnqseq doesn't have tfa, which I remember to be associated with b.subtilis network.